src/app/app.component.less

```diff
@@ -3,5 +3,5 @@ p {
 }
 
 multi-line-chart {
-  width: 500px;
```

Comment:
forgot about this guy. Interesting question is -- how did you arrive at 800 px, and what assumption is the D3 code using to calculate the relative x position over the chart?

Reply:
MultiLineChartComponent#ngOnInit has a local variable `width`!